src/api/MainApiService.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable class-methods-use-this */
+import { RequestRefreshToken, RequestRegisterType, RequestRecoveryPasswordType } from './helpers/ApiRequestTypes';
+import { MAIN_API_URLS } from './helpers/ApiConstant';
+import { servicesGet, servicesPost, servicesPut } from './helpers/ApiHelpers';
+
+
+class MainApiService {
+
+  userRegister (params:RequestRegisterType):Promise<unknown> {
+    return servicesPost(MAIN_API_URLS.USER.REGISTER, {params})
```

Comment:
Think about adding try...catch block for handle errors. Just proposal. If you have another plans to catching errors in another place - just say about this.